src/pages/RFOX/components/RewardsAndClaims/Rewards.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, CardBody } from '@chakra-ui/react'
+import { TxStatus } from '@shapeshiftoss/unchained-client'
+import { useMemo } from 'react'
+import { rune } from 'test/mocks/assets'
+import { TransactionsGroupByDate } from 'components/TransactionHistory/TransactionsGroupByDate'
+import { selectTxIdsByFilter } from 'state/slices/selectors'
+import { useAppSelector } from 'state/store'
+
+type RewardsProps = {
+  headerComponent: JSX.Element
+}
+
+export const Rewards = ({ headerComponent }: RewardsProps) => {
+  const filter = useMemo(() => ({ txStatus: TxStatus.Confirmed, assetIdFilter: rune.assetId }), [])
+  const txIds = useAppSelector(state => selectTxIdsByFilter(state, filter))
+  const limitTxIds = useMemo(() => {
+    return txIds.slice(0, Number(5))
```

Comment:
Are we going to eventually support loading more or only last 5?

Reply:
Idk I'm pretty sure we will need a pagination at one point but this is not something we did spec + I'm not sure how we will want to handle this at this time

Reply:
Can we add a TODO here to accomodate this?

Reply:
Also changed the key in [683d93f](https://github.com/shapeshift/web/pull/7098/commits/683d93fbc32c413ab11ded7c3829d3d96e3720c9)